api/include/mfxstructures.h

```diff
@@ -2269,6 +2269,48 @@ MFX_PACK_END()
 
 #endif
 
+#if (MFX_VERSION >= MFX_VERSION_NEXT)
+/* Multi-adapters Querying structs */
+typedef enum
+{
+    MFX_ENCODE_COMPONENT = 1,
```

Comment:
From my perspective `MFX_COMPONENT_<name>` style will be better.

Reply:
Well, maybe